src/source_sampling.cpp

```diff
@@ -28,14 +37,23 @@ void pyne::particle_birth_(double* rands,
                            double* y,
                            double* z,
                            double* e,
-                           double* w) {
+                           double* w,
+                           int* c) {
     std::vector<double> rands2(rands, rands + 6);
-    std::vector<double> samp = sampler->particle_birth(rands2);
-    *x = samp[0];
-    *y = samp[1];
-    *z = samp[2];
-    *e = samp[3];
-    *w = samp[4];
+//    std::vector<double> samp = sampler->particle_birth(rands2);
+    sampler->particle_birth(rands2);
+    //*x = samp[0];
```

Comment:
These commented lines can probably be removed (here and in other areas of the code)